packages/grafana-ui/src/themes/default.ts

```diff
@@ -5,23 +5,23 @@ const theme: GrafanaThemeCommons = {
   typography: {
     fontFamily: {
       sansSerif: "'Roboto', Helvetica, Arial, sans-serif",
-      serif: "Georgia, 'Times New Roman', Times, serif",
       monospace: "Menlo, Monaco, Consolas, 'Courier New', monospace",
     },
     size: {
-      base: '13px',
-      xs: '10px',
-      s: '12px',
-      m: '14px',
-      l: '18px',
+      root: '100%',
+      base: '.8125rem',
```

Comment:
would prefer pixel base, as it makes things o much easier.

Reply:
The problem with pixels is the user can't change the font size. I was also thinking of using rem for margin and padding forcing us to use .25rem, .5rem, .75rem, 1rem, 1.5rem. Making things aligned. I don't think rem will be much harder than pixels.